ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -24,6 +24,19 @@ scp "${SSHOPTS[@]}" /usr/bin/openshift-tests /usr/bin/kubectl "root@${IP}:/usr/l
 # Tests execution
 set +e
 
+# Run tests in disconnected mode
+LOCAL_TEST_IMAGE_REPO=${LOCAL_REGISTRY_DNS_NAME}:${LOCAL_REGISTRY_PORT}/localimages/local-test-image
+
+ssh "${SSHOPTS[@]}" "root@${IP}" bash - << EOF
+
+set -xeuo pipefail
+
+source ~/dev-scripts/common.sh
+openshift-tests images --to-repository ${LOCAL_TEST_IMAGE_REPO} > /tmp/mirror
```

Comment:
How big are all the test images? Just wondering if /tmp is the right place.

Reply:
`/tmp/mirror` it's just a small plain file with the name of the images prefixed with the target registry name. I will recheck, but I had the impression that `oc mirror ...` will not touch the tmp folder but upload directly the specified images into the target registry